addons/kommander/0.x/kommander-1.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: kommander
+  labels:
+    kubeaddons.mesosphere.io/name: kommander
+    # TODO: we're temporarily supporting dependency on an existing default storage class
+    # on the cluster, this hack will trigger re-queue on Addons until one exists.
+    # This was originally added to support the PVC's needed for the Kubecost subcomponent.
+    kubeaddons.mesosphere.io/hack-requires-defaultstorageclass: "true"
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.5.0-3"
+    appversion.kubeaddons.mesosphere.io/kommander: "1.5.0-beta.0"
+    endpoint.kubeaddons.mesosphere.io/kommander: /ops/portal/kommander/ui
+    appversion.kubeaddons.mesosphere.io/thanos: 0.3.21
```

Comment:
It looks like it's getting it's version from here to create a file this is why the 0.x directory is being created

Reply:
oh weird